packages/expo/src/provider/ClerkProvider.tsx

```diff
@@ -6,13 +6,15 @@ import * as WebBrowser from 'expo-web-browser';
 import type { TokenCache } from '../cache/types';
 import { isNative, isWeb } from '../utils/runtime';
 import { getClerkInstance } from './singleton';
+import type { BuildClerkOptions } from './singleton/types';
 
 export type ClerkProviderProps = React.ComponentProps<typeof ClerkReactProvider> & {
   /**
    * The token cache is used to persist the active user's session token. Clerk stores this token in memory by default, however it is recommended to use a token cache for production applications.
    * @see https://clerk.com/docs/quickstarts/expo#configure-the-token-cache-with-expo
    */
   tokenCache?: TokenCache;
+  passkeys?: BuildClerkOptions['passkeys'];
```

Comment:
Also let's add a comment here that mentions that this is experimental feature etc.